src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -61,6 +63,7 @@ protected override void Initialize()
             IServiceProvider serviceProvider = this;
 
             this.sonarAnalyzerManager = new SonarAnalyzerManager(serviceProvider);
+            this.suppressionManager = new SuppressionManager(serviceProvider);
```

Comment:
The _SuppressionManager_ class wires all other bits together so it's probably a good place to start to get an overview of how everything hangs together.